providers/summary/clearlydefined.js

```diff
@@ -340,6 +344,43 @@ class ClearlyDescribedSummarizer {
       `${get(result, 'described.urls.registry')}/archive/${coordinates.revision}.zip`
     )
   }
+
+  addDebData(result, data, coordinates) {
+    setIfValue(result, 'described.releaseDate', extractDate(data.releaseDate))
+    if (!data.registryData) return
+    setIfValue(result, 'described.urls.registry', this.getDebianRegistryUrl(data.registryData))
+    setIfValue(result, 'described.urls.version', this.getDebianRegistryUrl(data.registryData))
+    const architecture = coordinates.revision.split('_')[1]
+    const downloadUrl = new URL(
+      'http://ftp.debian.org/debian/' + data.registryData.find(entry => entry.Architecture === architecture).Path
+    ).href
+    setIfValue(result, 'described.urls.download', downloadUrl)
+  }
+
+  addDebSrcData(result, data) {
+    setIfValue(result, 'described.releaseDate', extractDate(data.releaseDate))
+    if (!data.registryData) return
+    setIfValue(result, 'described.urls.registry', this.getDebianRegistryUrl(data.registryData))
+    setIfValue(result, 'described.urls.version', this.getDebianRegistryUrl(data.registryData))
+    const downloadUrl = new URL(
+      'http://ftp.debian.org/debian/' + data.registryData.find(entry => entry.Path.includes('.orig.tar.')).Path
+    ).href
+    // There is also patches URL which is related to sources but it's not part of the schema
+    setIfValue(result, 'described.urls.download', downloadUrl)
+  }
+
+  getDebianRegistryUrl(registryData) {
+    const registryPath = registryData[0].Path
+    if (registryPath) {
+      // Example: ./pool/main/0/0ad/0ad_0.0.17-1.debian.tar.xz -> http://ftp.debian.org/debian/pool/main/0/0ad
```

Comment:
Is this example correct? Should the result have the file on the end?

Reply:
Yes, it's supposed to be a URL leading to a human readable website page. Unfortunately, with Debian the registry/package manager doesn't really exist, so that's the closest page we can get.